todoapp/todos/models.py

```
@@ -1,18 +1,22 @@
+    user = models.ForeignKey(settings.AUTH_USER_MODEL , on_delete=models.CASCADE)
+    name = models.CharField(max_length=1000)
+    done = models.BooleanField(default=False)
+    date_created = models.DateTimeField(default=timezone.now)
+    date_completed = models.DateTimeField(blank=True , default=None , null=True)
 
-        Add string representation for this model with todos name.
-    """
+    def __str__(self) -> str:
+        return self.name
+    
+    def save(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)